spec/lib/mas/model_geocoder_spec.rb

```diff
@@ -0,0 +1,81 @@
+RSpec.describe ModelGeocoder do
+  let(:model_class) do
+    Class.new do
+      attr_accessor :address_line_one, :address_line_two, :address_postcode, :longitude, :latitude
+
+      def update_coordinates!(*args); end
+
+      def full_street_address
+        [address_line_one, address_line_two, address_postcode, 'United Kingdom'].reject(&:blank?).join(', ')
+      end
+    end
+  end
+
+  let(:model) do
+    model_class.new.tap do |thing|
+      thing.address_line_one = address_line_one
+      thing.address_line_two = address_line_two
+      thing.address_postcode = address_postcode
+    end
+  end
+
+  let(:address_line_one) { '120 Holborn' }
+  let(:address_line_two) { 'London' }
+  let(:address_postcode) { 'EC1N 2TD' }
+  let(:expected_coordinates) { [51.5180697, -0.1085203] }
+
+  describe '#geocode' do
+    context 'when the model address can be geocoded' do
+      it 'returns the coordinates' do
+        VCR.use_cassette('geocode-one-result') do
```

Comment:
I do indeed. Held off yesterday to chat about `:geocode!` then promptly forgot 😀